.stylelintrc.json

```diff
@@ -8,13 +8,13 @@
     "scss/at-rule-no-unknown": true,
     "scss/dollar-variable-default": true,
     "scss/dollar-variable-first-in-block": [true, { "ignore": ["comments", "imports"] }],
-    "scss/dollar-variable-pattern": "^(example|utrecht)-[a-z0-9-]+$",
-    "scss/percent-placeholder-pattern": "^(example|utrecht)-[a-z0-9-]+$",
+    "scss/dollar-variable-pattern": "^(basis|example|utrecht)-[a-z0-9-]+$",
```

Comment:
Do we need these variable patterns (or stylelintrc) for the infrastructure repository? I wouldn't immediately expect there to be some css?

Reply:
Probably not, but we'll move this config file to an npm package for https://github.com/nl-design-system/basis/issues/9